pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -134,6 +149,7 @@ <h2 class="my-4 font-bold">2. ORIGINATOR DETAILS</h2>
         </div>
 
         {{ $originatorNationalIdentification := $originator.NationalIdentification }}
+        {{ if $originatorNationalIdentification }}
```

Comment:
Same thing here but with national identification. Here we're resolving the rendering error and below listing the fields in case users want to provide the information.